app/models/manageiq/providers/redhat/infra_manager/event_parser.rb

```diff
@@ -69,18 +69,23 @@ def self.parse_new_target(full_data, message, ems)
 
     {
       :ems_id         => ems.id,
-      :vm             => parse_new_vm(full_data[:vm], message),
+      :vm             => parse_new_vm(full_data[:vm], message, name),
       :cluster        => parse_new_cluster(cluster_ref, cluster[:id], cluster_name),
       :resource_pools => parse_new_rp(cluster[:id], cluster_name),
       :folders        => parse_new_dc(full_data[:data_center])
     }
   end
 
-  def self.parse_new_vm(vm, message)
+  def self.parse_new_vm(vm, message, name)
     ems_ref = ManageIQ::Providers::Redhat::InfraManager.make_ems_ref(vm[:href])
 
+    msg = message.split(/\s/)[1]
+    if %w(NETWORK_ADD_VM_INTERFACE NETWORK_INTERFACE_PLUGGED_INTO_VM).include?(name)
```

Comment:
No, only event type would give use the info. Message is a human readable string from an audit log. I need it to parse vm name.